client/src/views/landing.vue

```diff
@@ -6,7 +6,17 @@
           <Stats class="pr-10"/>
         </v-col>
         <v-col cols="12" class="mb-n2">
-          <span class="h2 colour-white">Name Request <sup class="beta-tag">Beta</sup></span>
+          <span class="h2 colour-white">Name Request
+            <v-tooltip bottom nudge-right="10" content-class="bottom-tooltip" transition="fade-transition">
+              <template v-slot:activator="{ on }">
+                <sup class="beta-tag" v-on="on">Beta</sup>
+              </template>
+              <p>The Name Request website is available as a Beta version. Name Requests obtained through the Beta are
+                official name requests and can be used in the province of British Columbia.</p>
+              <p>As part of a new <u>agile software development process</u>, the Name Request website will be
```

Comment:
👍

Reply:
Yea, this is actually pretty helpful info!